### DISCUSSION TITLE
AMD RX470 8gb Miner 1.26 Crash

### DISCUSSION THREAD

**Discussion by neo4uo** [Lolliedieb/lolMiner-releases#495] [Q&A]
I keep getting crashes either within 5 minutes or 15 minutes. The software runs great but then closes all of a sudden. I have looked thru the documentation and was trying to get the log to work and have been unable to get the log to work to see why the miner keeps crashing. I am using the RX470 8GB cards on the 1.26 version. I have tried adding the --log =on switch to bat file and still no log file. If anything can someone explain the full syntax to enable the log file so I can see what is going on to stop the software?

Thanks

**Comment by MiroslavKatic** [comment]
There is no "--log=on" or "--log=off" param, help is misleading unfortunately.
Only 1 or 0 are valid params: "--log 1 --logfile filename.txt"
But log file won't help you, I have the same problem with RX 580 (mining BEAM), log file won't provide what is the problem.
For now, task scheduler is the only real solution, restart lolminer process every 60 minutes.
